celer/homotopy.py

```diff
@@ -153,6 +155,13 @@ def celer_path(X, y, eps=1e-3, n_alphas=100, alphas=None,
 
         alpha = alphas[t]
         t0 = time.time()
+
+        if scale_gap:
+            if np.any(y):
+                tol = tol / np.linalg.norm(y)
```

Comment:
If y has a huge norm (primal objs are large), we use a smaller tolerance. I think it should be the opposite